src/components/Layout/EditorSidebar/EditorSidebar.js

```diff
@@ -119,6 +71,47 @@ function EditorSidebar(props) {
     ].includes(key)
 
     const isYesNo = key === QUESTION_TYPES.YES_NO
+    //configuration for certain types of questions
+    const choicesConfiguration = [
```

Comment:
You moved it *again*?

Reply:
Maybe move to separate constants?

Reply:
No, because for every new question we should generate unique id for configurations, when I moved it out, id for every item in question configurations has the same id, so it should be here